### DISCUSSION TITLE
Clawback Not Working - 3 Days Has Not "Clawed Back" Stays in Pending Balance

### DISCUSSION THREAD

**Discussion by NewSunSEO** [Chia-Network/chia-blockchain#16764] [Support]
Hello, I was testing the new claw back feature the other night.  I did not approve the transaction & let it revert to the clawback feature.  It has been sitting in my "Pending balance" for about 3 days now.  It just says pending with the same amount I sent.  How can I get this clawback to complete?

Thank you

**Comment by BrandtH22** [comment]
Hey @NewSunSEO , as a heads up we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia)

If you are running the latest version of chia (2.1.1) then you can run the below commands in PowerShell (windows) or terminal (mac/linux) to force the clawback transaction:

1. Identify the clawback transaction id by running `chia wallet get_transactions --clawback`
2. Use the transaction id in this command and for windows run chia rpc wallet spend_clawback_coins '{\"coin_ids\":[\"TRANSACTION_ID\"], \"force\":true, \"fee\":0}'` | for linux run `chia rpc wallet spend_clawback_coins '{"coin_ids":["TRANSACTION_ID"], "force":true, "fee":0}'`

Make sure to replace the TRANSACTION_ID in the second command with the resulting transaction id from the first command.

Give it some time and you should see the clawback in the wallet.

**Reply by NewSunSEO** [reply]
Thank you for your reply.  I entered that & nothing seemed to change, but I went & resynched the wallet & it gave me a new option to clawback the transaction, which was successful.  Thank you for your help!

![image](https://github.com/Chia-Network/chia-blockchain/assets/11369854/aa6f02e4-b448-44a2-b072-4eebb3bf05ff)

**Reply by BrandtH22** [reply]
Very glad to hear it was successful and the funds are where they are intended.
Let us know if other questions arise.

I am going to close this ticket but feel free to reopen it or join us on discord if any other questions arise (https://discord.gg/chia)
